pom.xml

```diff
@@ -199,7 +199,7 @@
         <module>spring</module>
         <module>guice</module>
         <module>weld</module>
-        <module>openejb</module>
+<!--        <module>openejb</module>-->
```

Comment:
I've pushed deprecating OpenEJB forward to V5.x so we can do JPMS in V6.

https://github.com/cucumber/cucumber-jvm/issues/1763